code/modules/clothing/clothing.dm

```diff
@@ -370,6 +376,7 @@
 	icon_override = 'icons/mob/clothing/head.dmi'
 	body_parts_covered = HEAD
 	slot_flags = SLOT_FLAG_HEAD
+	dyeable = FALSE
```

Comment:
This is already `FALSE` by default on `/obj/item/clothing` right? I think we should not be setting false here then.

Reply:
updated